docs/source/configuration.rst

```diff
@@ -247,7 +249,8 @@ Defaults to NONE for the default template. ::
 
 placement_group
 """""""""""""""
-Cluster placement group. The can be one of three values: NONE, DYNAMIC and an existing placement group name. When DYNAMIC is set, a unique placement group will be created as part of the cluster and deleted when the cluster is deleted.
+Cluster placement group. The can be one of three values: NONE, DYNAMIC and an existing placement group name. When DYNAMIC is set, a unique placement group will be created as part of the cluster and deleted when the cluster is deleted. Note that when
+using a placement group only certain EC2 instance types are allowed to be used for the master and compute nodes.
```

Comment:
Add `... used, see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/placement-groups.html for a list of instance types that can be used within a placement group.`